.github/workflows/Build.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     strategy:
       matrix:
         cmd:
-          - gradle testShardHibernate -i --scan
+          - gradle testShardHibernate -i --scan --no-parallel
```

Comment:
Setting this to ` --no-parallel` explicitly otherwise the `parallel` setting I've added in gradle.properties makes these tests run in parallel and they'll fail.